src/NUnitEngine/nunit.engine.tests/Services/TestRunnerFactoryTests/Results/Net20ThreeItemExpectedRunnerResults.cs

```diff
@@ -114,34 +108,22 @@ private static RunnerResult GetProcessModelInProcessResult(DomainUsage domainUsa
                 case DomainUsage.None:
                     return new RunnerResult
                     {
-                        TestRunner = typeof(AggregatingTestRunner),
-                        SubRunners = new[]
-                        {
-                            new RunnerResult { TestRunner = typeof(LocalTestRunner) },
-                            new RunnerResult { TestRunner = typeof(LocalTestRunner) },
-                            new RunnerResult { TestRunner = typeof(LocalTestRunner) }
-                        }
```

Comment:
I think this change is also incorrect, there should be an Aggregating runner here, right?

Reply:
(This is only reinforcing my view that was should have a specific `LocalAggregatingTestRunner` - to distinguish the different needs here! 😅)

Reply:
I would say no. LocalTestRunner derives from DirectTestRunner, which is capable of aggregating the result of multiple drivers. In this case, the result would be the same, whether using one runner with three drivers or three runners plus an aggregating runner. However, I think the latter is better because it's simpler and also conforms to the only acceptable solution when using DomainUsage.Single.

The key is that we only want an aggregating runner when we are aggregating runners. :-) And we only need separate runners when the settings require it (as for running in separate AppDomains).